programs/marginfi/src/state/marginfi_account.rs

```diff
@@ -87,15 +87,30 @@ pub enum BalanceDecreaseType {
 }
 
 #[derive(Copy, Clone)]
-pub enum WeightType {
+pub enum RequirementType {
     Initial,
     Maintenance,
     Equity,
 }
 
+impl RequirementType {
+    /// Get oracle price type for the requirement type.
+    ///
+    /// Initial and equity requirements use the time weighted price feed.
+    /// Maintenance requirement uses the real time price feed, as its more accurate for triggering liquidations.
+    pub fn get_oracle_weight_type(&self) -> OraclePriceWeightType {
```

Comment:
let's call this get_oracle_price_type to match struct renaming (this fn name is actually the one that confused me originally, when called below)

Reply:
Ok will do!